src/app/ReadClient.h

```diff
@@ -339,7 +339,8 @@ class ReadClient : public Messaging::ExchangeDelegate
     FabricIndex mFabricIndex            = kUndefinedFabricIndex;
     InteractionType mInteractionType    = InteractionType::Read;
     Timestamp mEventTimestamp;
-    EventNumber mEventMin = 0;
+    EventNumber mEventMin             = 0;
+    bool mIsAttributeReportIBsPresent = false;
```

Comment:
Please document the semantics of this member.  It's really not clear from the name what it means....

And maybe it should be named `mSawAttributeReportsInCurrentReport` or something.

Reply:
mSawAttributeReportsInCurrentReport is a better name and is self-explained.